index.bs

```diff
@@ -162,7 +162,7 @@ NOTE: This means that {{XRSessionMode/"immersive-ar"}} sessions support all the
 The following code checks to see if {{XRSessionMode/"immersive-ar"}} sessions are supported.
 
 <pre highlight="js">
-navigator.xr.supportsSession('immersive-ar').then(() => {
+navigator.xr.isSessionSupported('immersive-ar').then(() => {
```

Comment:
`isSessionSupported()` resolves to a boolean which needs to be checked.

```suggestion
navigator.xr.isSessionSupported('immersive-ar').then((supported) => {
  if (!supported) { return; }
```

Reply:
Ah, yes. haven't read example other than function name...